code/dynamic_programming/src/fibonacci/fibonacci.cpp

```diff
@@ -0,0 +1,22 @@
+#include <iostream>
+using namespace std;
+
+int fib(int n)
+{
+    int *ans = new int[n + 1]; //creating an array to store the outputs
+    ans[0] = 0;
+    ans[1] = 1;
+    for (int i = 23; i <= n; i++)
+    {
+        ans[i] = ans[i - 1] + ans[i - 2];  // storing outputs for further use
+    }
+    return ans[n];
+}
+int main()
+{
+    int n;
+    cout << " Enter the Number";  // taking input
```

Comment:
Remove the leading space and add semicolon at the end for better output display
`cout << "Enter the Number: ";`

Reply:
ok sir